Clerk stopped working on Java 1.8 with 0.8.463 (2022-06-16)

Hi,

it appears that Clerk has stop working with Java 1.8 with the latest release. 

Although Java 1.8 is a very old version, it is still supported by Clojure as the baseline, and there are still a lot of dev envs out there that are still using Java 1.8 as their main development env (for stability continuation reasons).

To reproduce switch to java 1.8 and try to load Clerk:

```
Exception in thread "main" Syntax error compiling at (nextjournal/markdown.clj:1:1).
        at clojure.lang.Compiler.load(Compiler.java:7652)
...
Caused by: java.lang.UnsupportedClassVersionError: org/graalvm/polyglot/Context has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
        at java.lang.ClassLoader.defineClass1(Native Method)
...
```

This is due to switching to a newer version of `org.graalvm.js/js` (from `21.3.0` to `22.1.0` ) with this checkin: https://github.com/nextjournal/markdown/commit/512301a7f255bc2488798afc2a9b619b3c16c6b3#diff-551f87f7577a57c2f4f74b35056d5c7985331f7a38976810fd2bbc213bfe7bb5L5.

It appears that [21.3](https://www.graalvm.org/release-notes/21_3/) was the last branch with support of Java 1.8
```
The 21.3 release line will be the last one to contain GraalVM Enterprise based on Oracle JDK 8. 22.x and further will only contain distributions based on Oracle Java 11 and 17.
```

Is there any appetite to keep supporting users who for whatever reason are still on 1.8 by reverting  to the 21.3 revision of graaljs?

Alternative, the user can replace the dependency in the project file with something as follows, but this might as well stop working on later clerk version (somewhere in `deps.edn`):
``` edn
:aliases {:dev {:override-deps {org.graalvm.js/js {:mvn/version "21.3.0"}}}}
```

Of course they could always upgrade to the a newer java environment, but this is always cumbersome to switch or work on two jave envs at the same time.

Thanks

--- a/deps.edn
+++ b/deps.edn
@@ -8,7 +8,7 @@
 
         com.nextjournal/beholder {:mvn/version "1.0.0"}
 
-        io.github.nextjournal/markdown {:mvn/version "0.4.116"}
+        io.github.nextjournal/markdown {:mvn/version "0.4.121"}
 
         com.taoensso/nippy {:mvn/version "3.1.1"}
         mvxcvi/multihash {:mvn/version "2.0.3"}

--- a/resources/viewer-js-hash
+++ b/resources/viewer-js-hash
@@ -1,1 +1,1 @@
-Wv8hGV7cmRqw3oaUZ1wRL4qCfvw
+3hoZ5UZEWazyJPnUcTvAbLMmp6zJ

